ts/features/itwallet/lifecycle/store/reducers/index.ts

```diff
@@ -0,0 +1,30 @@
+import { getType } from "typesafe-actions";
+import { Action } from "../../../../../store/actions/types";
+import { itwLifecycleStateUpdated } from "../actions";
+
+export enum ItwLifecycleState {
+  "ITW_LIFECYCLE_INSTALLED",
+  "ITW_LIFECYCLE_OPERATIONAL",
+  "ITW_LIFECYCLE_VALID",
+  "ITW_LIFECYCLE_DEACTIVATED"
+}
+
+const initialState: ItwLifecycleState =
+  ItwLifecycleState.ITW_LIFECYCLE_INSTALLED;
```

Comment:
Question: can't we start from `OPERATIONAL`? What's the difference? Is it related to the wallet instance?

Reply:
Exactly, that was the idea. I think it might be useful to track a state where the wallet instance was never activated (`INSTALLED`), for example to skip checking whether the wallet instance was revoked at app start up, or other similar checks.

Reply:
Makes sense to me. Can we already provide a more refined condition on how we manage these states? For example, in order to assert that the wallet is in an `INSTALLED` state, should we check this part of the reducer but also the absence of an integrity key tag? However, the remaning states can't be refined right now.

Let me know what you think.

Reply:
I agree and pushed a small modification.

Reply:
I added comments to the enum to better explain each state.